utils/start.sh.sample

```diff
@@ -49,18 +50,13 @@ if [[ $IRIS_ENABLED = true ]]; then echo "$IRIS_NAMESPACE/iris:$IRIS_VERSION"; f
 if [[ $SOFTWARE_ENABLED = true ]]; then echo "$RABBITMQ_NAMESPACE/rabbitmq:$RABBITMQ_VERSION"; fi
 if [[ $SOFTWARE_ENABLED = true ]]; then echo "$SLURM_NAMESPACE/slurm:$SLURM_VERSION"; fi
 if [[ $SOFTWARE_ENABLED = true ]]; then echo "$SOFTWAREROUTER_NAMESPACE/software_router:$SOFTWAREROUTER_VERSION"; fi
-echo ""
+echo "create server ssh Keys"
 
 # Create server SSH keys if needed
-SERVER_SSHKEYS_FILE="$SERVER_SSHKEYS_PATH/id_rsa"
-if [ ! -e $SERVER_SSHKEYS_FILE ]
-then
-    if [[ "$OSTYPE" == "linux-gnu" ]]; then
-        apt-get install -y ssh-keygen
-    fi
-    ssh-keygen -t rsa -N "" -C $CORE_URL -f $SERVER_SSHKEYS_FILE
-fi
+SERVERLOCAL="slurm"
+SERVER_SSHKEYS_FILEPATH="$SERVER_SSHKEYS_PATH/slurm"
 
+mkdir SERVER_SSHKEYS_FILEPATH
```

Comment:
It is a bit painful but the current philosophy for directories creation is to give the responsibility of directories creation to the Cytomine installer. Look for other paths in the configuration (IMAGES_PATH, ...) which are never created. 
Long answer: in some installations, these paths come from disks mounted on the network, and their creation isn't so simple.